peachjam/js/components/FindDocuments/AdvancedSearch.vue

```diff
@@ -0,0 +1,223 @@
+                id="flynote"
+                name="flynote"
+                type="text"
+                class="form-control"
+                :aria-describedby="$t('Flynote')"
+                :placeholder="$t('Search documents by flynote')"
+                :value="modelValue.flynote"
+                @input="onChange"
+              >
+            </div>
+          </div>
+
+          <div class="col-12 mb-3">
+            <div class="form-group">
+              <label for="content">{{ $t('Content:') }}</label>
+              <textarea
+                id="content"
+                name="content"
+                class="form-control"
+                :aria-describedby="$t('Content')"
+                :placeholder="$t('Search documents by content')"
+                :value="modelValue.content"
+                @input="onChange"
+              />
+            </div>
+          </div>
+          <div class="col-6 mb-3">
+            <div class="form-group">
+              <label for="date_from">{{ $t("Date from:") }}</label>
+              <input
+                id="date_from"
+                name="date_from"
+                type="date"
+                class="form-control"
+                :aria-describedby="$t('Date from:')"
+                :placeholder="$t('Enter start date')"
+                :value="modelValue.date.date_from"
+                :disabled="disableDate"
+                @change="onDateChange"
+              >
+            </div>
+          </div>
+          <div class="col-6 mb-3">
+            <div class="form-group">
+              <label for="date_to">{{ $t("Date to:") }}</label>
+              <input
+                id="date_to"
+                name="date_to"
+                type="date"
+                class="form-control"
+                :aria-describedby="$t('Date to:')"
+                :placeholder="$t('Enter end date')"
+                :value="modelValue.date.date_to"
+                :disabled="disableDate"
+                @change="onDateChange"
+              >
+            </div>
+          </div>
+          <div
+            v-if="invalidDateRange"
+            class="col-12 mb-3 text-danger"
+          >
+            {{ $t('The date range you have selected is invalid. Please choose a correct date range.') }}
+          </div>
+        </div>
+        <div class="text-end">
+          <button
+            type="submit"
+            class="btn btn-primary"
+          >
+            {{ $t('Search') }}
+          </button>
+        </div>
+      </form>
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'AdvancedSearch',
+  props: {
+    modelValue: {
+      type: Object,
+      default: () => ({})
+    },
+    globalSearchValue: {
+      type: String,
+      default: ''
+    }
+  },
+  emits: ['submit', 'update:modelValue', 'global-search-change'],
+
+  computed: {
+    invalidDateRange () {
+      if (!this.modelValue.date.date_from && !this.modelValue.date.date_to) {
```

Comment:
I think it should be allowed to only enter one date. eg. "everything from X onwards" and "everything up to Y".